src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -59,18 +69,27 @@ public void launchTask(final ExecutorDriver driver, final TaskInfo taskInfo) {
           .setTaskId(journalNodeTask.taskInfo.getTaskId())
           .setState(TaskState.TASK_RUNNING)
           .build());
+      healthCheckJN = new TimedHealthCheck(driver, journalNodeTask);
+      timer.scheduleAtFixedRate(healthCheckJN, schedulerConf.getHealthCheckWaitingPeriod(),
+          schedulerConf.getHealthCheckFrequency());
```

Comment:
Major issue: I thought that http or command health checks are exactly what we want here?!? You can use http to see if a NN/DN/JN/ZN web endpoint at a certain port returns a valid response (e.g. 200); and you could use a command (e.g. script) to test if you can bind to a specific port, which is exactly what you're doing here. Please explain the difference.
I'm strongly in favor of reverting this patch until we can do health checks the standard Mesos way. I'm fine with deferring this out of Q1, but I don't want to have to support a hacky health check that doesn't support the features of a normal Mesos task health check.